DEVELOPMENT.md

```diff
@@ -36,12 +36,67 @@ $ LIBGIT2_FORCE=1 make libgit2
 ```
 
 **Note:** Example shown is for Arch Linux, but likewise procedure can be
-followed using any other package manager, e.g. `apt`.
+followed using any other package manager. Some distributions may have slight 
+variation of package names (e.g. `apt install -y cmake openssl libssh2-1-dev`).
 
 ## How to run the test suite
 
-You can run the unit tests by simply doing
+The test suite depends on [envtest] being installed. For minimum required 
+version refer to the variable `ENVTEST_BIN_VERSION` in the [Makefile](./Makefile).
+
+You can run the unit tests by simply doing:
 
 ```bash
 make test
 ```
+
+[envtest]: https://book.kubebuilder.io/reference/envtest.html#installation
+
+
+## How to run the controller locally
+
+Install flux on your test cluster:
+
+```sh
+flux install
+```
+
+Scale the in-cluster controller to zero:
+
+```sh
+kubectl -n flux-system scale deployment/source-controller --replicas=0
+```
+
+Run the controller locally:
+
+```sh
+make run
+```
+
+## How to install the controller
+
+Set the name of the container image to be created from the source code. This will be used 
+when building, pushing and referring to the image on YAML files.
+
+```sh
```

Comment:
The alternative shorter way of the below multi-step instruction is the following:

```console
$ IMG=container-registry.com/yourhandle/source-controller TAG=latest BUILD_ARGS=--push \
make docker-build
```

Reply:
Nice one, I added this as well as the longer version.